client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -3,65 +3,50 @@
  */
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { reduxForm } from 'redux-form';
 import { localize } from 'i18n-calypso';
 import { flowRight } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import CompactCard from 'components/card';
-import FormFieldset from 'components/forms/form-fieldset';
-import FormButton from 'components/forms/form-button';
-import FormLabel from 'components/forms/form-label';
 import HeaderCake from 'components/header-cake';
-import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
-import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
-import SectionHeader from 'components/section-header';
-import { getSelectedSiteSlug } from 'state/ui/selectors';
-
-const form = 'extensions.zoninator.newZone';
-
-const ZoneCreator = ( { siteSlug, translate } ) => (
-	<div>
-		<HeaderCake backHref={ `/extensions/zoninator/${ siteSlug }` }>
-			{ translate( 'Add a zone' ) }
-		</HeaderCake>
-
-		<form>
-			<SectionHeader label={ translate( 'New zone' ) }>
-				<FormButton compact>{ translate( 'Save' ) }</FormButton>
-			</SectionHeader>
-			<CompactCard>
-				<FormFieldset>
-					<FormLabel htmlFor="zoneName">{ translate( 'Zone name' ) }</FormLabel>
-					<ReduxFormTextInput name="zoneName" />
-				</FormFieldset>
-
-				<FormFieldset>
-					<FormLabel htmlFor="zoneDescription">{ translate( 'Zone description' ) }</FormLabel>
-					<ReduxFormTextarea name="zoneDescription" />
-				</FormFieldset>
-			</CompactCard>
-		</form>
-	</div>
-);
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import ZoneDetailsForm from '../../forms/zone-details-form';
+import { addZone } from '../../../state/zones/actions';
+
+const ZoneCreator = ( {
+	saveZone,
+	siteId,
+	siteSlug,
+	translate,
+} ) => {
+	const save = ( form, data ) => saveZone( siteId, form, data );
+
+	return (
+		<div>
+			<HeaderCake backHref={ `/extensions/zoninator/${ siteSlug }` }>
+				{ translate( 'Add a zone' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **14** times:
`translate( 'Add a DNS Zone' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/zoninator-new-zone-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).